_utility/send_gh_comment.sh

```diff
@@ -12,9 +12,9 @@ REPO_OWNER=$1
 REPO_NAME=$2
 PR_NUMBER=$3
 AUTH_TOKEN=$4
-BUCKET_NAME=$(_utility/normalize_bucket_name.sh $5)
+DEPLOYMENT_URL=$5
 
-COMMENT="Deployed: $BUCKET_NAME"
+COMMENT="Deployed: $DEPLOYMENT_URL"
 LAST_COMMENT_ID=$(curl https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/issues/$PR_NUMBER/comments?access_token=$AUTH_TOKEN -X GET | jq '[.[] | select(.user.login=="CoduranceBot")][0].id')
```

Comment:
Minor: either format the line by using `\` to split the lines or use functions for each separate functionality - will help readability